<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mini_page</class>
 <widget class="QWidget" name="mini_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#settings{
	background-color: rgba(255,228,196, 0.4);
}
*{
	background-color: transparent;
	color: #fff8ea;
	border: none;
	font-size: 16px;
	font-weight: 700;
	font-family: ZCOOL KuaiLe;
}

/*歌词标签*/
#lrc_lab{
	color: #CF467F;
	font-size: 21px;
	font-weight: 800;
}
/*时间标签*/
[time]{
	color: #DE76A3;
	font-size: 16px;
}
/*按钮样式*/
QPushButton:hover{
	color: white;
	margin: 2px;
}
#close_btn{
	border-image: url(:/title/tools/resource/bg/icon/title/clear.png);
}
#play_btn{
	border-image: url(:/play/tools/resource/bg/icon/play/play.png);
}
/*进度滑动条*/
QSlider#music_time_slider{
	padding-top: 2px;
	margin: 3px;
}
QSlider#music_time_slider::groove 
{
	padding-left: -2px;
	padding-right: -2px; 
	background:white;
	height:6px;
	border-radius: 3px;
}

QSlider#music_time_slider::sub-page 
{
	background-color: #5494DE;
	background-color: #FF75B1;
	border-radius: 3px;
}

QSlider#music_time_slider::add-page 
{
	background-color: #78AAE3;
	background-color: #F2A7C6;
	border-radius: 3px;
}

QSlider#music_time_slider::handle    
{
	height: 10px;  
	width: 10px;
	border-radius: 5px;
	background: #5C89BD;
	background: #E84D90;
	margin-top: -2px;
	margin-bottom: -2px;
}
QSlider#music_time_slider::handle:hover
{
	background: white;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>288</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="close_btn">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lrc_lab">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>音乐下载路径：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="title" stdset="0">
      <string>-1</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0,0,1">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="play_btn">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="save" stdset="0">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MusicTimeSlider" name="music_time_slider">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="now_time_lab">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="time" stdset="0">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicTimeSlider</class>
   <extends>QSlider</extends>
   <header>PySide_UI.ui.tools.base.musictimeslider</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>close_btn</sender>
   <signal>clicked()</signal>
   <receiver>mini_page</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
